pkg/commands/run.go

```diff
@@ -358,11 +358,13 @@ func (e *Executor) releaseFileLock() {
 
 //nolint:gomnd
 func initRunFlagSet(fs *pflag.FlagSet, cfg *config.Config) {
-	// Config file config
-	rc := &cfg.Run
-	initConfigFileFlagSet(fs, rc)
+	fs.BoolVar(&cfg.InternalCmdTest, "internal-cmd-test", false, wh("Option is used only for testing golangci-lint command, don't use it"))
+	if err := fs.MarkHidden("internal-cmd-test"); err != nil {
+		panic(err)
+	}
+
+	// --- Output config
```

Comment:
Not a huge fan of these divider comments, should `initRunFlagSet` rather be split into multiple functions, one for each config section?

Reply:
I added the divider only to help me on the next step.
If possible I would prefer to keep the divider because it will impact my current branch.
In the next step, all the flag sets will be revamped.

Reply:
Go for it, keep it here for now!